index.md

```diff
@@ -0,0 +1,48 @@
+---
+# Feel free to add content and custom Front Matter to this file.
+# To modify the layout, see https://jekyllrb.com/docs/themes/#overriding-theme-defaults
+
+layout: home
+---
+
+{:.cryostat-heading-1}
+Cryostat 1.0.0
+
+Create and manage JDK Flight Recordings to troubleshoot your containerized Java applications
+
+{:.cryostat-heading-2}
+Now available for Kubernetes 1.19+ and OpenShift 4.6+
+
+## Installing Cryostat Operator
+Coming soon to [OperatorHub](https://operatorhub.io/). In the meantime, you can install
+the Cryostat Operator using kubectl, or by deploying the bundle image with Operator SDK.
+
+### Install with kubectl
+```
+$ kubectl create namespace cryostat-operator-system
+$ kubectl apply -k github.com/cryostatio/cryostat-operator//config/default?ref=v1.0.0
```

Comment:
Is this URL correct? The `//` looks off at least, but maybe it's intended.

Also my `kubectl` has no `-k`, but my `oc` does - not sure if this is an error in the instruction or if I have an out of date `kubectl`.

Reply:
kubectl added `-k` in 1.14, it's specifically for Kustomize directories. Maybe yours is from the origin-clients 3.11 Fedora package?

The `//` tells Kustomize/kubectl to only download the directory rooted after the `//`. Since our Kustomize directory is self-contained, it's a bit more effecient to download just that. Details: https://github.com/kubernetes-sigs/kustomize/blob/master/examples/remoteBuild.md#url-format